src/main/java/ru/hogwarts/school/controller/InfoController.java

```diff
@@ -17,4 +19,12 @@ public int getPort() {
         return port;
     }
 
+    @GetMapping("/step_4")
+    public int stepFour() {
+        return Stream.iterate(1, a -> a+1)
+                .limit(1_000_000)
+                .parallel()
+                .reduce(0, Integer::sum);
```

Comment:
также можно заменить на sum()
```suggestion
                .sum();
```